integration_tests/androidx_test/src/main/res/layout/espresso_activity.xml

```diff
@@ -23,6 +23,8 @@
       android:id="@+id/text_view"
       android:layout_height="wrap_content"
       android:layout_width="wrap_content"
+      android:letterSpacing="0.05"
+      android:textScaleX="1.5"
```

Comment:
What about adding another `TextView` with negative `textScaleX`?

Reply:
Sorry, I don't notice it. We can keep it.

Reply:
To test negative value of textScaleX. You have created test for negative testScaleX at  ShadowPaintTest, it's better if we can test it too with real resource mechanism .

Reply:
Seems like I was wrong and described behaviour with the same width but other direction works only for `Button` but not for usual `TextView`.
I will probably delete it and use 0 for negative `textScaleX` values for measuring text.

Reply:
Fixed.
Also added new Espresso test with `TextView` and negative `textScaleX`.